configs/templates/neptune-dev/spack.yaml

```diff
@@ -8,7 +8,8 @@ spack:
   definitions:
   - compilers: ['%aocc', '%apple-clang', '%gcc', '%intel', '%oneapi']
   - packages:
-      - neptune-env +espc +python +xnrl ^esmf@8.7.0b11 snapshot=b11
+      - neptune-env +espc        ^esmf@8.7.0b11 snapshot=b11
```

Comment:
I think 8.7.0 was released, should we use that instead of the snapshot?

Reply:
I'll have to go back and update all the stacks, which is doable. But first I need to get esmf 8.7.0 from spack develop into spack-stack. This will add another several days. Do you want to wait for that? I am fine either way.

Reply:
Thinking about this a bit more, I would like to have a follow-up PR that removes some of the auto-generated spack-stack modules to trim down the list of loaded modules even more (for both neptune-env and neptune-python-env). We'll hvae to roll out a new version of spack-stack on all platforms for that anyway, therefore I'll defer the official esmf@8.7.0 release to that rollout.